src/index.html

```diff
@@ -0,0 +1,240 @@
+      </p>
+      <table>
+        <thead>
+          <tr>
+            <th scope="col">Item</th>
+            <th scope="col">Marker Numbers</th>
+            <th scope="col">Marker Size (mm)</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Arena boundary</td>
+            <td>0 - 3</td>
+            <td>100</td>
+          </tr>
+        </tbody>
+      </table>
+      <p>
+        The markers can be printed on a black-and-white printer.
+      </p>
+      <p>
+        Eight arena wall markers are positioned around the arena as shown in the <a href="#arena-specification">arena specification</a>. Marker codes are mirrored on each side of the arena.
+      </p>
+      <p>
+        Wall markers are placed such that the bottom of the marker is 20 ±5mm from the arena floor.
+      </p>
+      <h3><a class="anchor" id="arena-specification"></a>Arena</h3>
+      <img src="./fig-arena.svg" alt="Arena diagram" width="90%" height="auto">
+      <ol>
+        <li>
+          The above figure represents, to scale, a birds eye view of the arena. The arena is
+          <strong>5.4m</strong> square.
+        <li>
+          The floor of the arena is covered with a closed-loop, short pile carpet.
+        <li>
+          The perimeter of the arena floor is delimited by the arena wall, which has a minimum height of 100mm.
+        <li>
+          Starting areas are 1000 ±50mm squares and will be marked by tape which is considered part of the starting area.
+        <li>
+          Starting area 0 is at the top of the arena diagram, and area 1 is at the bottom.
+        <li>
+          The green boxes denote a robot in their start locations, shown in yellow. Robots may start anywhere inside their starting area facing any direction.
+        <li>
+          The inner walls of the arena have a minimum height of 170mm.
+        <li>
+          SourceBots reserves the right to have match officials in the arena during games.
+      </ol>
+      <h3><a class="anchor" id="flag-specification"></a>Robot flags</h3>
+      <img src="./fig-flag.svg" alt="Robot flag" width="40%" height="auto">
+      <ol>
+        <li>
+          A "robot flag" is a removable identifier that will be attached to a robot throughout a match. It features identifying areas to allow spectators to easily associate a robot with its starting area. The markings in the identifying areas are intentionally not specified.
+        <li>
+          Flags are not counted when considering the starting size of the robot.
+        <li>
+          The identifying part of the robot flag must be visible when attached to the mount.
+        <li>
+          The mounting cylinder must be permanently affixed to the main chassis of a robot, and vertical when the robot is in its typical stopped position.
+      </ol>
+      <h3><a class="anchor" id="can-specification"></a>Cans</h3>
+      <ol>
+        <li>
+          Cans are cylindrical metallic containers weighing 300g ±20g.
```

Comment:
You need to account for the metal of the can
```suggestion
          Cans are cylindrical metallic containers weighing 350g ±20g.
```